ipa-core/src/protocol/ipa_prf/oprf_padding/distributions.rs

```diff
@@ -139,7 +139,7 @@ impl Distribution<i32> for DoubleGeometric {
 /// Truncated Double Geometric distribution.
 #[derive(Debug, PartialEq)]
 pub struct TruncatedDoubleGeometric {
-    shift_doubled: u32, // move 2 * shift to constructor instead of sample
+    pub shift_doubled: u32, // move 2 * shift to constructor instead of sample
```

Comment:
Would it be worth providing a method like `TruncatedDoubleGeometric::mean` so this doesn't need to be public? Or perhaps even if the parameters are public, to encapsulate code related to the distribution? It does seem reasonable for the distribution parameters to be public, but if they are it might make more sense to do so consistently for all of them.

Reply:
Yes, I made the function `mean_and_std_bound` in `insecure.rs` to do this.  I think that was when I needed to make this parameter public.   I think moving some code around is a good follow up step and there would move lots much of this code from `insecure.rs` into this `distributions.rs` file and then also not need to make this param public.